src/arrayMethodSort.js

```diff
@@ -3,9 +3,41 @@
 /**
  * Implement method Sort
  */
+function campareAsStrings(firstElement, secondElement) {
+  const a = String(firstElement);
+  const b = String(secondElement);
+
+  if (a > b) {
+    return 1;
+  }
+
+  if (a < b) {
+    return -1;
+  }
+
+  return 0;
+}
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = campareAsStrings) {
+    let count;
```

Comment:
I dont know why but I got confused by this variable name.If I understand your code, count is  variable checking if some strings changed indexes. Also you are checking just >0 statement so maybe use boolean type. 
Its just my thoughts.

Reply:
That's fine, imho.